lib/engines/content_object_store/test/integration/content_block_editions_test.rb

```diff
@@ -24,18 +24,19 @@ class ContentBlockEditionsTest < ActionDispatch::IntegrationTest
     document_title = "Some Title"
     block_type = "block_type"
     details = {
-      foo: "Foo text",
-      bar: "Bar text",
+      "foo" => "Foo text",
```

Comment:
This syntax is what comes back out of the database so we switch our assertion to match.